src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -152,11 +155,14 @@ function findPathMatchedNodes(navNode, pathString, depth) {
         return Boolean(r.path)
       })
       if (routesWithPaths.length) {
-        const inferredPathParts = routesWithPaths[0].path.split('/')
-        inferredPathParts.pop()
-        const inferredPath = inferredPathParts.join('/')
-        if (inferredPath == pathString) {
-          return [{ title: navNode.title }]
+        // Iterate over all child nodes, match may not be in first node
+        for (let i = 0; i < routesWithPaths.length; i++) {
```

Comment:
This iteration is necessary because on https://developer.hashicorp.com/vault/docs/get-started/developer-qs, we have links in the sidebar that _start_ their lives as external `href` links but end up with `path` values that are _external to docs_. This messes with the expectation that `{ title, path }` nodes will be internal to docs, which was previously true.

I didn't realize this initially, which is why I took the initial approach of traversing nav-data (in `getFallbackTitle`) if a `title` match can't be found.